tests/components/device_automation/test_init.py

```diff
@@ -21,7 +21,7 @@ def entity_reg(hass):
     return mock_registry(hass)
 
 
-def _same_triggers(a, b):
+def _same_dicts(a, b):
```

Comment:
This only tests that the keys are the same, not the values.

Reply:
Fixed.